lib/app_profiler/backend/vernier_backend.rb

```diff
@@ -43,7 +43,9 @@ def start(params = {})
         @mode = params.delete(:mode) || DEFAULTS[:mode]
         raise ArgumentError unless AVAILABLE_MODES.include?(@mode)
 
-        @metadata = params.delete(:metadata)
+        if Gem.loaded_specs["vernier"].version < Gem::Version.new("1.7.0")
```

Comment:
In 1.7.0 or greater, we can just pass it directly as as it was added in https://github.com/jhawthorn/vernier/pull/145/files